spec/controllers/shops_controller_spec.rb

```diff
@@ -15,4 +15,25 @@
     get :index
     expect(response.body).to have_content(invisible_distributor.name)
   end
+
+  it 'renders distributed properties' do
+    duplicate_property = create(:property, presentation: 'dairy')
+    producer = create(:supplier_enterprise, properties: [duplicate_property])
+    property = create(:property, presentation: 'dairy')
+    product = create(:product, properties: [property])
+    producer.supplied_products << product
+
```

Comment:
If you want to doing a smoke test here, maybe it is more appropriate to test that producer and product properties are covered, than checking that there are no duplicates.

Reply:
The absence of duplicates is already covered in the serializer specs (and it does work) but I added individual tests for product and producer properties in the controller as well.